public/pages/Dashboard/utils/helpers.test.js

```diff
@@ -0,0 +1,848 @@
+          end_time: null,
+          agg_alert_content: {
+            parent_bucket_path: 'composite_agg',
+            bucket_keys: ['groupBy1KeywordValue', 'groupBy2KeywordValue'],
+            bucket: {
+              doc_count: 10,
+              field1: {
+                value: 100,
+              },
+              field2: {
+                value: 200,
+              },
+              key: {
+                groupBy1: 'groupBy1Keyword',
+                groupBy2: 'groupBy2Keyword',
+              },
+            },
+          },
+        },
+        {
+          id: 'bucket-monitor-trigger1-alert2',
+          monitor_name: 'bucket-monitor',
+          trigger_id: 'bucket-monitor-trigger1-id',
+          trigger_name: 'bucket-monitor-trigger1',
+          state: 'ACTIVE',
+          start_time: 1638360000000,
+          end_time: null,
+          agg_alert_content: {
+            parent_bucket_path: 'composite_agg',
+            bucket_keys: ['groupBy1KeywordValue', 'groupBy2KeywordValue'],
+            bucket: {
+              doc_count: 10,
+              field1: {
+                value: 100,
+              },
+              field2: {
+                value: 200,
+              },
+              key: {
+                groupBy1: 'groupBy1Keyword',
+                groupBy2: 'groupBy2Keyword',
+              },
+            },
+          },
+        },
+        {
+          id: 'bucket-monitor-trigger2-alert1',
+          monitor_name: 'bucket-monitor',
+          trigger_id: 'bucket-monitor-trigger2-id',
+          trigger_name: 'bucket-monitor-trigger2',
+          state: 'ACTIVE',
+          start_time: 1638361800000,
+          end_time: null,
+          agg_alert_content: {
+            parent_bucket_path: 'composite_agg',
+            bucket_keys: ['groupBy1KeywordValue', 'groupBy2KeywordValue'],
+            bucket: {
+              doc_count: 10,
+              field1: {
+                value: 100,
+              },
+              field2: {
+                value: 200,
+              },
+              key: {
+                groupBy1: 'groupBy1Keyword',
+                groupBy2: 'groupBy2Keyword',
+              },
+            },
+          },
+        },
+        {
+          id: 'query-monitor-trigger1-alert1',
+          monitor_name: 'query-monitor',
+          trigger_id: 'query-monitor-trigger1-id',
+          trigger_name: 'query-monitor-trigger1',
+          state: 'ACTIVE',
+          start_time: 1638360000000,
+          end_time: null,
+        },
+        {
+          id: 'query-monitor-trigger2-alert1',
+          monitor_name: 'query-monitor',
+          trigger_id: 'query-monitor-trigger2-id',
+          trigger_name: 'query-monitor-trigger2',
+          state: 'ACTIVE',
+          start_time: 1638361800000,
+          end_time: null,
+        },
+      ];
+      const expectedOutput = [
+        {
+          ACTIVE: 2,
+          ACKNOWLEDGED: 0,
+          ERROR: 0,
+          total: 2,
+          alerts: [
```

Comment:
Seems like `expectedOutput` can be defined using `alerts`, right? Would be good to reuse constants here and reduce duplicate lines of code.

Reply:
The alerts can be defined as constants, then in individual tests fill them the `alerts` array

Reply:
I see. Thanks for clarifying!